python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -516,18 +485,23 @@ async def enable_devices(self):
                 commands.CameraCableWrapPower(on=True),
                 # commands.CameraCableWrapEnableTracking(on=True),
             ]
-            await self.send_commands(*enable_commands)
+            await self.send_commands(*power_on_commands)
+        except salobj.ExpectedError as e:
```

Comment:
Not sure I understand what is the gain from catching and treating `salobj.ExpectedError` differently.  Since you are raising the exception again afterwards, couldn't you do without it?

Reply:
Good point. I was logging an exception with the non-ExpectedError version, but it's not an interesting exception so I'll take your advice.